model/item.cc

```diff
@@ -161,8 +161,14 @@ namespace minsky
  
   bool Item::visible() const 
   {
+	if (auto i=dynamic_cast<const IntOp*>(this))
+	  if (i->intVar->attachedToDefiningVar()) return false; 
```

Comment:
Isn't an intVar a defined variable by definition?

Reply:
If i->coupled(), then the intop remains visible, which is why I added this condition.